content/open-source/oci/what-are-oci-artifacts.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "What are OCI Artifacts?"
+type: "article"
+description: "OCI Artifacts are a way of using OCI registries, or container registries that are compliant with specifications set by the Open Container Initiative, to store arbitrary files."
+date: 2022-06-09T15:22:20+01:00
+lastmod: 2022-06-09T15:22:20+01:00
+draft: false
+images: []
+menu:
+  docs:
+    parent: "oci"
+weight: 150
+toc: true
+---
+
+[OCI artifacts](http://github.com/opencontainers/artifacts) are a way of using OCI registries, or container registries that are compliant with specifications set by the [Open Container Initiative](https://edu.chainguard.dev/open-source/oci/what-is-the-oci/), to store arbitrary files. While container registries were originally designed to store and distribute images, container engineers are increasingly using them to store non-container files such as Helm charts, Tekton bundles, and policy modules. By storing associated objects in the same infrastructure as containers, container engineers can consolidate their security and management efforts. Another benefit of using OCI registries is that provide a [content-addressable API](https://edu.chainguard.dev/open-source/oci/what-is-the-oci/#image-manifest), a way of referring to files that assures their authenticity and integrity.  
```

Comment:
same as jason's comment, we shouldnt ever link to https://github.com/opencontainers/artifacts